examples/bri-3/src/bri/zkp/components/merkleTree/merkleTree.ts

```diff
@@ -0,0 +1,74 @@
+import { Tree } from "./tree"
+import { IMerkleTree} from "./merkleTree.interface";
+import { PreciseProofs } from "ew-precise-proofs-js";
```

Comment:
could you please explain usage of this package and why it will be useful for us, currently it is just for merkle tree creation, but i assume that will change?

Reply:
@skosito we will be storing zkp witnesses in merkle trees. also we will use merkle trees to record BPI state associated with a BPI account. similar to how ethereum does it ... one could ask the question if we should use Patricia-Merkle trees vs regualr merkle trees but for now this is ok, IMHO.